packages/snap-preact-components/src/components/Molecules/RadioList/readme.md

```diff
@@ -11,42 +11,42 @@ Renders a list of options with radio inputs.
 The required `options` prop specifies an array of Option Objects to be rendered.
 
 ```jsx
-<RadioSelect options={store.options} />
+<RadioList options={store.options} />
 ```
 
 ### native
 The `native` prop will use native html `<input type='radio'>` elements.
 
 ```jsx
-<RadioSelect options={store.options} native={true} />
+<RadioList options={store.options} native={true} />
 ```
 
 ### hideRadios
 The `hideRadios` prop will disable the radio elements from rendering.
 
 ```jsx
-<RadioSelect options={store.options} hideRadios={true} />
+<RadioList options={store.options} hideRadios={true} />
 ```
 
 ### titleText
 The `titleText` prop is will render a title element
 
 ```jsx
-<RadioSelect options={store.options} titleText={'Per Page'} />
+<RadioList options={store.options} titleText={'Per Page'} />
 ```
 
 ### disabled
 The `disabled` prop will put the inputs in a disabled state.
 
 ```jsx
-<RadioSelect options={store.options} disabled={true} />
+<RadioList options={store.options} disabled={true} />
 ```
 
 ### selected
 The `selected` prop specifies the currently selected Option object. 
 
 ```jsx
-<RadioSelect options={store.options} selected={store.pagination.pageSize} />
+<RadioList options={store.options} selected={store.pagination.pageSize} />
```

Comment:
Similar to `List` component comment - may want to use the `store.pagination.pageSizeOptions` for the options here.